client/views/actions/topic_set_by.tpl

```diff
@@ -0,0 +1 @@
+Topic set by <a href="#" class="user" data-name="{{nick}}">{{mode}}{{nick}}</a> on {{localeDate when}}
```

Comment:
Argh, I see. Kind of a pain that they come as separate events, but oh well, we'll see later I guess as the result is kind of odd at the moment.